gnomad/utils/vcf.py

```diff
@@ -796,6 +876,85 @@ def make_info_dict(
             },
         }
         info_dict.update(popmax_dict)
+    elif grpmax:
+        grpmax_dict = {
+            f"{prefix}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AC{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
```

Comment:
do we just use Number A for everything even if only one value is expected?

Reply:
Honestly I just kept what was already here for popmax.

Reply:
Though from the docs, I think this makes sense 
```
• A: The field has one value per alternate allele. The values must be in the same order as listed in the ALT
column (described in section 1.6).
• R: The field has one value for each possible allele, including the reference. The order of the values must be the
reference allele first, then the alternate alleles as listed in the ALT column.
```
Since grpmax is the highest alt allele frequency